src/ResourceManager/DataLakeStore/Commands.DataLakeStore.Test/Commands.DataLakeStore.Test.csproj

```diff
@@ -121,7 +125,18 @@
   <ItemGroup>
     <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>
+    <Reference Include="Microsoft.Azure.Management.DataLake.Store, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\Microsoft.Azure.Management.DataLake.Store.2.4.1-preview\lib\net452\Microsoft.Azure.Management.DataLake.Store.dll</HintPath>
+    </Reference>
+    <Reference Include="Microsoft.Rest.ClientRuntime, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
Yes, I added this back for debugging. Forgot to remove afterwards.
Currently getting this error for only `TestAdlsVirtualNetworkRules` : `Message:System.Management.Automation.ActionPreferenceStopException : The running command stopped because the preference variable "ErrorActionPreference" or common parameter is set to Stop: TestManagementClientHelper class wasn't initialized with the NetworkManagementClient client.`
I can only find this error message from [here](https://github.com/Azure/azure-sdk-tools/blob/aadb7c9829dbf8ac09b18675c41b40d52ef3f57f/src/Common/Commands.Common.Test/Mocks/MockClientFactory.cs#L67). But looks like the [build](https://azuresdkci.westus2.cloudapp.azure.com/job/powershell/8044/) failed for different reasons. Thanks!